app/controllers/all/local_authorities/projects_controller.rb

```diff
@@ -14,8 +14,8 @@ def show
   end
 
   private def projects_for_local_authority(local_authority_code)
+    EstablishmentsFetcherService.new(Project.not_completed).batched!
```

Comment:
I feel like this is just fetching the establishments for no reason becuse the next line gets another set of projects from the DB and passes that along - I am not 100% on this though.

Regardless, I think it's good enough for now - this whole prefetching thing is fast becoming a real issue that we should take a step back and try to solve.

Reply:
That's pretty much what it's doing 😆